src/client/src/apps/MainRoute/layouts/AppLayout.tsx

```diff
@@ -12,11 +12,15 @@ export interface Props {
 }
 
 const AppLayout: React.FC<Props> = ({ before, header, body, footer, after }) => {
+  // TODO - move to openfin-platform
+  //@ts-ignore
+  const isChildView = window.fin && window.fin.me && window.fin.me.isView
+
```

Comment:
These lines shouldn't be here, they are OpenFin Platform API specific, and should move to that module.